plugins/com.google.cloud.tools.eclipse.appengine.compat/src/com/google/cloud/tools/eclipse/appengine/compat/cte13/CloudToolsEclipseProjectNotifier.java

```diff
@@ -145,6 +145,7 @@ private IStatus upgradeProjects(Collection<IProject> projects, SubMonitor progre
       IStatus result = CloudToolsEclipseProjectUpdater.updateProject(project, progress.newChild(1));
       status.merge(result);
     }
-    return status;
+    // rewrite if OK as otherwise Progress View shows the updating projects message
```

Comment:
project's

Reply:
In this case it's literally "Updating projects for …"